frontends/concrete-python/Makefile

```diff
@@ -16,7 +16,10 @@ RUNTIME_LIBRARY=${COMPILER_BUILD_DIRECTORY}/lib/libConcretelangRuntime.so
 venv:
 	$(PYTHON) -m venv .venv
 	. .venv/bin/activate
-
+ifeq (,$(wildcard ${RUNTIME_LIBRARY}))
+	$(PIP) install concrete-python
+# $(PIP) install --extra-index-url https://pypi.zama.ai/cpu concrete-python==X.Y.ZdevYYYYMMDD # uncomment to use a nightly
```

Comment:
Maybe could be great to have an environment variable (CONCRETE_VERSION?=latest) to select else the latest public version of concrete-python, le latest dev one or a specific one?

Reply:
I wasn't able to have the last dev version. You have to put the exact version you want. Maybe we don't configure well package on our side. I adding an en var.